src/main/scala/io/qbeast/spark/delta/QbeastStatsUtils.scala

```diff
@@ -29,7 +29,7 @@ import com.fasterxml.jackson.module.scala.ClassTagExtensions
 import io.qbeast.core.model.mapper
 import io.qbeast.core.model.QbeastStats
 
-object QbeastStatsUtils {
+private[delta] object QbeastStatsUtils {
```

Comment:
Again here, maybe name it `QbeastDeltaStatsUtils`.
Although I do not see any dependency on `delta` here, I guess it's because it is the way to codify the stats on the Delta Format.  


Perhaps we should have an interface (trait) on module `core` that holds the stats serialization methods and other capabilities that each format-specific implementation should develop.

Reply:
This was a decision made with Cesare in a previous PR, where the new QbeastStats trait is serialized to/from JSON in Delta, but maybe not in other formats. So for now, we are placing this utility file here

Reply:
Cool for me 👍